draft-ietf-ppm-dap.md

```diff
@@ -1461,6 +1461,16 @@ The Helper responds to the Leader with HTTP status code 201 Created and a body
 consisting of the `AggregationJobResp`, with media type
 "application/dap-aggregation-job-resp".
 
+Changing an aggregation job's parameters is illegal, so further requests to
+`PUT /tasks/{tasks}/aggregation_jobs/{aggregation-job-id}` for the same
+`aggregation-job-id` but with a different `AggregationJobInitReq` in the body
+MUST fail with an HTTP client error status code.
+
+Additionally, it is not possible to rewind or reset the state of an aggregation
+job. Once an aggregation job has been continued at least once (see
+{{agg-continue-flow}}), further requests to initialize that aggregation job MUST
+fail with an HTTP client error status code.
```

Comment:
Can you check to make sure this doesn't contradict {{aggregation-step-skew-recovery}}?

Reply:
It does not. That section concerns itself with the rounds of the continuation phase of the aggregation sub-protocol. This paragraph is saying you can't go back from the continuation phase to the initialization phase. See [discussion with @branlwyd](https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/pull/471#discussion_r1229862877) for context.